compiler/src/dotty/tools/dotc/reporting/diagnostic/messages.scala

```diff
@@ -1770,4 +1770,21 @@ object messages {
           |"""
   }
 
+  case class EnumCaseDefinitionInNonEnumOwner(owner: Symbol)(implicit ctx: Context)
+    extends Message(EnumCaseDefinitionInNonEnumOwnerID) {
+      val kind = "Syntax"
+      val msg = em"case not allowed here, since owner ${owner} is not an `enum' object"
+      val explanation = {
+
+        hl"""
+            | ${"case"} is only allowed at this place if the surrounding object is the companion object of an `enum class`.
```

Comment:
Just curious, is `${"case"}` equivalent to `case`? I see similar uses above which I don't understand.